tests/pending/run/t6260-delambdafy.check

```diff
@@ -1,4 +0,0 @@
-f(C@2e)
```

Comment:
They were moved, not deleted, but the files changed too much for git to detect the move.